### PR TITLE
Create new drawer navigator design 

### PR DESCRIPTION
**Pull request by pectom**

### CODE REVIEW

File: src/modules/navigation/AppDrawerNavigator.js [hunk 1]
```diff
@@ -1,43 +1,109 @@
...
+      screen: WhatToTakeView,
+      navigationOptions: {
+        drawerIcon: (
+          <Image
+            source={require('../../../assets/icons/backpack.png')}
+            style={ksStyle.drawerIcon}
+          />
+        ),
+        drawerLabel: 'Co zabrać?',
+      },
```
Comment (mblasiak) [inline src/modules/navigation/AppDrawerNavigator.js]: Imo we can define those objects elsewhre, so the Navigator can stay nice and compact

File: src/modules/navigation/AppDrawerNavigator.js [hunk 2]
```diff
@@ -1,43 +1,109 @@
...
+            source={require('../../../assets/icons/partners.png')}
+            style={ksStyle.drawerIcon}
+          />
+        ),
         drawerLabel: 'Partnerzy',
-        drawerIcon: ({ tintColor }) => (
-          <IconComponent
-            name="people"
-            size={ksStyle.icon.size}
-            color={tintColor}
```
Comment (mblasiak) [inline src/modules/navigation/AppDrawerNavigator.js]: As above

File: src/modules/navigation/AppDrawerNavigator.js [hunk 3]
```diff
@@ -1,43 +1,109 @@
...
 import DashboardScreen from '../../views/dashboardScreen/DashboardScreen';
-import AppBottomNavigator from './AppBottomNavigator';
+import { Image } from 'react-native';
 import { ksStyle } from '../../styles/basic/ksBasic';
+import { DrawerItem } from '@react-navigation/drawer';
 
-const IconComponent = MaterialIcons;
-
-export default createDrawerNavigator(
+export default new DrawerNavigator(
```
Comment (mblasiak) [inline src/modules/navigation/AppDrawerNavigator.js]: Imo it's to big and shroud be spitted a bit. It's hard to read in current form

File: src/modules/navigation/AppDrawerNavigator.js (hunk 1 above)
Comment (pectom) [reply src/modules/navigation/AppDrawerNavigator.js]: You're right. I will put config in different file.

**Comment by AnetaPociecha** [comment]
Imo everything looks great. We can merge this branch to master. The flaw is that the code should be refactored to avoid prettier errors. There is separate task for eslint errors so I can take care of it once branch is merged.

**Pull request merged**